From 39c661ee11b6d199befcc28ef10867b55620dfac Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Wed, 22 Jul 2020 16:37:01 -0700
Subject: [PATCH 21/47] Update Bahrain as per article in The National
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

* backzone (Asia/Bahrain): Adjust transitions before 1944 to
match Ashleigh Stewart’s article in today’s The National (Abu Dhabi).
---
 backzone | 17 ++++++++++++++++-
 1 file changed, 16 insertions(+), 1 deletion(-)

diff --git a/backzone b/backzone
index 91fa21d..7cf026d 100644
--- a/backzone
+++ b/backzone
@@ -459,7 +459,22 @@ Zone	Asia/Aden	2:59:54	-	LMT	1950
 			3:00	-	+03
 
 # Bahrain
-Zone	Asia/Bahrain	3:22:20 -	LMT	1920     # Manamah
+#
+# From Paul Eggert (2020-07-22):
+# Most of this data comes from:
+# Stewart A. Why Gulf Standard Time is far from standard: the fascinating story
+# behind the time zone's invention. The National (Abu Dhabi). 2020-07-22.
+# https://www.thenational.ae/arts-culture/why-gulf-standard-time-is-far-from-standard-the-fascinating-story-behind-the-time-zone-s-invention-1.1052589
+# Stewart writes that before 1940 some companies in Bahrain were at +0330 and
+# others at +0323.  Reginald George Alban, a British political agent based in
+# Manama, worked to standardize this, and from 1941-07-20 Bahrain was at
+# +0330.  However, BOAC asked that clocks be moved to gain more light at day's
+# end, so Bahrain switched to +04 on 1944-01-01.
+#
+# We don't know when companies chose +0330 or +0323 before 1940; for now
+# assume that there was no real standard in Bahrain before 1941-07-20.
+Zone	Asia/Bahrain	3:22:20 -	LMT	1941 Jul 20  # Manamah
+			3:30	-	+0330	1944 Jan  1
 			4:00	-	+04	1972 Jun
 			3:00	-	+03
 
-- 
1.8.3.1

